backend/src/GraphQl/Institutions/ChangeInstitutionOperatingStateInput.cs

```diff
@@ -0,0 +1,7 @@
+using System;
+
+namespace Metabase.GraphQl.Institutions;
+
+public sealed record ChangeInstitutionOperatingStateInput(
```

Comment:
Same naming issue as above. It is not clear from this name that the new state is going to be `NOT_OPERATING`.